test/caching.bats

```diff
@@ -163,6 +163,36 @@ EOF
     [[ ! "${output}" =~ ^(.*found cached layer bind-test)$ ]]
 }
 
+@test "bind as struct" {
+    cat > stacker.yaml <<"EOF"
+bind-test:
+    from:
+        type: oci
+        url: ${{CENTOS_OCI}}
+    import:
+        - tree1/foo/bar
+    binds:
+        - Source: ${{bind_path}}
+          Dest: /root/tree2/foo
+    run: |
+        cp /stacker/foo /root/foobar1
+        cp /root/tree2/foo/bar /root/foobar2
+        ls /root
+EOF
+
+    mkdir -p tree1/foo
+    echo foo >> tree1/foo/bar
+    mkdir -p tree2/foo
+    echo bar >> tree2/foo/bar
+
+    bind_path=$(realpath tree2/foo)
+
+    # The layer should be built
+    stacker build --substitute bind_path=${bind_path} --substitute CENTOS_OCI=$CENTOS_OCI
+    out=$(stacker build --substitute bind_path=${bind_path} --substitute CENTOS_OCI=$CENTOS_OCI)
+    [[ "${out}" =~ ^(.*filesystem bind-test built successfully)$ ]]
```

Comment:
we do want a check for successful builds, but line 191 and 192 are running the build twice. remove 191.